src/faldbt/parse.py

```diff
@@ -72,8 +72,9 @@ def get_dbt_config(
         target=profile_target,
     )
 
-    if project_dir and not os.getenv("GITHUB_ACTIONS"):
+    if project_dir and not "PYTEST_CURRENT_TEST" in os.environ:
         # HACK: initializing dbt-fal requires cwd to be project_dir
+        # TODO: this doesn't work in pytest + Github Actions
```

Comment:
Could we consider somehow how to avoid the fact that

> initializing dbt-fal requires cwd to be project_dir


We could change the code in dbt-fal to check if it should cd into `project_dir` or understand where it is _standing_ right now better and not depend on that.

Reply:
Yes! I looked into that and we would have to somehow communicate project-dir to the dbt-fal code that's causing the error. The error happens because of this line: https://github.com/fal-ai/fal/blob/main/adapter/src/dbt/adapters/fal/impl.py#L39 , which is executed when the module is being imported.